flocker/node/script.py

```diff
@@ -157,6 +158,8 @@ def main(self, reactor, options, volume_service):
 
 def flocker_changestate_main():
     return FlockerScriptRunner(
+        # VolumeScript will supply an OpenStack volume service / pool
+        # to ChangeStateScript
```

Comment:
We can discuss in the meeting how the backend volume type will be selected. ie ZFS or Openstack block.